BUTTON_IDEAS.md

```diff
@@ -53,3 +53,16 @@ When adding ideas, please add them in the right category (or add a new one if yo
 | status  | Button name                    | Description                          | Submitted by                                  |
 | ------- | ------------------------------ | ------------------------------------ | --------------------------------------------- |
 | &#9744; | **Add Rounded Social Buttons** | A social button with rounded corners | [Neeraj Gupta](https://github.com/Neeraj3508) |
+
+## Hover Buttons
+
+| status  | Button name      | Description                                                      | Submitted by                                |
+| ------- | ---------------- | ---------------------------------------------------------------- | ------------------------------------------- |
+| &#9744; | **Sleek Button** | A sleek button whose background and outline change upon hovering | [Preity Priya](https://github.com/preity-p) |
```

Comment:
Can you please explain more what you mean by this button?

Reply:
when you hover over the button, the border gets pushed inside out, besides the opacity changing.

Reply:
Oh I see. Then please add this one to Animated as well and include this description instead of the one you added before as this one is more explanatory.

I'm still not sure about the name though. If you can find a better name for it then please change it and if not then that's fine.